proposals/LP0004-project-governance.md

```diff
@@ -0,0 +1,393 @@
+
+Members of the LLVM Foundation Board and employees of the LLVM Foundation are also members of the LLVM Community, as such they can also fill roles in any of the community structures defined in this proposal.
+
+
+### Modifications to the LLVM Decision Making Process
+
+The proposal process described in [LP0001 LLVM Decision Making](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md) should be adjusted in the following ways under this governance proposal.
+
+In step [3](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md?plain=1#L60) of the process, in addition to identifying review managers, the proposal will identify the relevant _area team_ to oversee the proposal. If there is no relevant _area team_, or if the decision spans across the domains of multiple _area teams_ the _project council_ will oversee the proposal.
+
+In steps [4](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md?plain=1#L61) and [8](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md?plain=1#L65), the roles designated to Chris Lattner are replaced by the overseeing _area team_ or the _project council_.
+
+## Voting Process
+
+Public notice of any vote will be published on the LLVM discussion forums under the "Elections" topic at least 14 days before voting opens with a schedule for the election. Each election will be open for 14 days. Reminders will be posted on the "Elections" topic at:
+
+* 7 days before voting opens.
+* The day voting opens.
+* 7 days after voting opens.
+* 48 hours before voting closes.
+* 24 hours before voting closes.
+
+Voter eligibility will be determined 7 days before voting opens by the rules for defining _voting contributors_. Voting will be executed using a private poll hosted by [Concordcet Internet Voting Service]([https://civs1.civs.us/](https://civs1.civs.us/)). Concordcet uses a ranked choice polling process.
+
+All eligible _voting contributors_ will be contacted via email with instructions on how to vote on the day voting opens.
+
+### Elections
+
+Elections for _area teams_ will be conducted in January of each year. A two week long nomination period will begin the second Monday in January. During the nomination period any _voting contributor_ can nominate themselves or another _voting contributor_ to run for any one _area team_. No individual can run for more than one _area team_ in a single election. Nominations will be recorded publicly for community visibility.
+
+Voting begins the fourth Monday in January and continues for 2 weeks. Election results will be announced no later than the second Monday in February. The term of the newly elected _area team_ begins the first Monday in March. Each _area team_ will meet during the first week in March to elect from themselves the team chair to re-constitute the _project council_.
```

Comment:
Is the intention here to imply that results are published immediately following 2 weeks of voting?
January can have either 4 or 5 Mondays, so some years this allows for a 1 week gap.

I think it should probably be considered that, in some years, the placement of this voting period can overlap significantly with extended holiday periods in Asia.  Which could foreseeably produce some engagement bias.

Reply:
Timing elections is tricky. My hope is that having a long enough voting period, and enough announcements before and during gives sufficient time for people to account for it. I'm also not at all opposed to moving the elections from January/February to a different time of the year.